install.sh

```diff
@@ -224,6 +252,44 @@ __install_mongodb() {
 	esac
 	__install_packages mongodb-org
 	_MONGO_INSTALLED=true
+
+}
+
+__configure_mongodb() {
+	printf "$_IMPORTANT Enabling MongoDB on startup. \n"
+	if [ $_OS = "Ubuntu" ]; then
+		if [ $_OS_CODENAME = "trusty" ]; then
+			# Ubuntu 14.04 uses Upstart for init
+			# https://www.digitalocean.com/community/tutorials/how-to-configure-a-linux-service-to-start-automatically-after-a-crash-or-reboot-part-1-practical-examples#auto-start-checklist-for-upstart
+			initctl stop service > /dev/null
+			initctl start service > /dev/null
+		else 
+			systemctl enable mongod.service > /dev/null
+			systemctl daemon-reload > /dev/null
+		fi
+	elif [ $_OS = "CentOS" ]; then
+		chkconfig mongod on > /dev/null
+	fi
+	printf "$_IMPORTANT Enabling MongoDB on startup process completed.\n"
+
+	printf "$_IMPORTANT Starting MongoDB. \n"
+	if [ $_OS = "Ubuntu" ]; then
+		if [ $_OS_CODENAME = "trusty" ]; then
+			# Ubuntu 14.04 uses Upstart for init
+			service mongod start > /dev/null
```

Comment:
initctl start not only enables the MongoDB service, it starts the starts the service. Here, we ask the service to be started again. This causes a crash and error: `start: job is already running: mongod`.

Just remove this line and put in a comment?

Reply:
There's probably no reason we can't just combine the enable and start checks anyway.  I'm not entirely sure anymore why there are separate in the first place.

Also, since I can't comment on the one above about replacing `service` with `mongod` here is my reaction: 🤦‍♂️  Good catch!